core/contracts/oracle/implementation/Voting.sol

```diff
@@ -536,12 +536,12 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
 
     /**
      * @notice Gets the queries that are being voted on this round.
-     * @return pendingRequests `PendingRequest` array containing identifiers
+     * @return activeRequests `PendingRequest` array containing identifiers
      * and timestamps for all pending requests.
      */
     // TODO(#969) Remove once prettier-plugin-solidity can handle the "override" keyword
     // prettier-ignore
-    function getPendingRequests() external override view returns (PendingRequest[] memory pendingRequests) {
```

Comment:
In `scripts/Voting.js`, `scripts/cli/admin.js`, `scripts/cli/voting/commitVotes`, `scripts/cli/voting/revealVotes`, `scripts/cli/voting/displayStatus` and `voter-dapp/src/ActiveRequests`

Reply:
I think I got all of these in the PR.